cmd/logoff.go

```diff
@@ -21,17 +21,13 @@ var logoffCmd = &cobra.Command{
 		if err != nil {
 			log.Fatalf("Failed to read configuration file. %s", err)
 		}
-		// Logoff the PAS REST API
-		err = client.Logoff()
-		if err != nil {
-			log.Fatalf("Failed to log off. %s", err)
-			return
-		}
 		// Remove the config file written to local file system
 		err = client.RemoveConfig()
 		if err != nil {
 			log.Fatalf("Failed to remove configuration file. %s", err)
 		}
+		// Logoff the PAS REST API
+		_ = client.Logoff()
```

Comment:
Is this intended to not catch error?

Reply:
Yes, because during testing I found that if a user was _already_ logged off somehow, it would not remove the config file.  So I have to bypass the logoff in order to ensure the config always is removed. At which point, it'll be okay if an error occurred to allow the user to naturally time out.